cmd/cluster-cloud-controller-manager-operator/main.go

```diff
@@ -112,9 +114,22 @@ func main() {
 		os.Exit(1)
 	}
 
+	platform := &platform.InfrastrucutreOwner{}
+
+	setupLog.Info("Waiting for platform objects")
+	if err = wait.PollImmediate(
```

Comment:
This should go away. We need to consider how to add and remove informers on managed resources dynamically during reconcile.